bootstrap.sh

```diff
@@ -141,6 +141,9 @@ do
 	    bootstrap_libtoolize ${LIBTOOL_BIN}ize${ltver}
 	    bootstrap automake$amver --foreign --add-missing --copy -f
 	    bootstrap autoconf$acver --force
+        # sometimes the autofiles in libltdl have a version mismatch.
+        # force-refresh them
+        (cd libltdl; bootstrap autoreconf -fi)
```

Comment:
As you know, bootstrapping in general is a risky step that often requires well-hidden prerequisites/dependencies. Are you sure this intrusion into libltdl space always safe? Is it possible that libltdl is incompatible with bootstrapping files a developer has installed for Squid? In other words, is it possible that Squid itself bootstraps just fine, but also bootstrapping libltld leads to problems?

What does libltld documentation say about bootstrapping libltdl itself?